packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -165,31 +94,65 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
   const context = useFluentContext();
   const { setStart, setEnd } = useTelemetry(Datepicker.displayName, context.telemetry);
   setStart();
+  const dateSelectionErrorString = 'A date selection is required';
   const datepickerRef = React.useRef<HTMLElement>();
-  const [open, setOpen] = React.useState<boolean>(false);
+  const [openState, setOpenState] = useAutoControlled<OpenState>({
```

Comment:
This should be just `boolean`

Reply:
@layershifter I use the enum to mimic the behavior of FabricUI/Teams. https://github.com/microsoft/fluentui/blob/0b666d9bc2e4b20e9267a78de4327998eda5f6b4/packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx#L204

Should I then simplify to boolean but lose the described behavior?

FYI: @pompomon

Reply:
I don't see any reason to have enum state, `open` is am `boolean` value

Reply:
@layershifter switched in [the following commit](https://github.com/microsoft/fluentui/pull/14563/commits/f4c038eeed968dfec4a00cc20d2194bc72072387).